backend/tests/restaurant/capacity.js

```diff
@@ -0,0 +1,84 @@
+import chai, { assert } from 'chai';
+import chaiHttp from 'chai-http';
+import config from '../../config';
+import connection from '../../database';
+chai.use(chaiHttp);
+var expect = chai.expect;
+
+
+before(async () => {
+  // Setup the database before any tests in this file run.
+  const errors = [];
+  await connection
+    .asyncQuery('INSERT INTO RESTAURANT VALUES (6,"KCF"),(7,"Mendat Ramen"),(8,"Nantoz"),(9,"Uni Zushi");')
+    .then(({ error }) => error && errors.push(error));
+
+  await connection
+    .asyncQuery(`INSERT INTO \`TABLE\` VALUES (1,6,1,4),(1,7,1,3),(1,8,1,4),(1,9,1,3),(2,6,3,8),(2,7,1,3),(2,8,4,10),(2,9,1,3),(3,6,2,6),(3,9,4,10),(4,6,1,4);`)
+    .then(({ error }) => error && errors.push(error));
+
+  assert.strictEqual(errors.length, 0, 'Expected no errors in initial setup');
+});
+
+
+// Testing restaurant/{restaurantID}/capacity end point
+describe('GET restaurant/{restaurantID}/capacity', () => {
+it('1. should return expected error if no restaurantID are entered', function(done) {
+    chai
+      .request(`${config.listen.address}:${config.listen.port}/restaurant`)
+      .get('/{restaurantID}/capacity')
+      .query({})
+      .end((err, res) => {
+        assert.isNull(err, 'Expected error to be null. Ensure an instance of the api is running');
+        const { body } = res;
+        assert.isArray(body, 'Expected response to contain a body object');
+        assert.deepEqual(body,[{minimum:null,maximum:null}],{error: '/restaurant/{restaurantID}/capacity GET endpoint needs a restaurantID query param'});
+        done();
+      });
+  });
+
+  it('2. should return expected error if incorrect parameters are entered', function(done) {
+    chai
+      .request(`${config.listen.address}:${config.listen.port}/restaurant`)
+      .get('/{restaurantID}/capacity')
+      .query({notRestaurantID:6})
```

Comment:
I'm not too sure how the testing framework works, but it seems like this line is passing a field named 'notRestaurantID', which is not a parameter that is accepted by the API. I think the field should still be named restaurantID while passing in a number that is not the ID of a restaurant

Reply:
I think this means no parameter input. then it will return null. 
But I can use one very large ID for this case if you insist.

Reply:
when no parameter input the route is    /restaurant//capacity